stapi_fastapi/main_router.py

```diff
@@ -15,14 +15,12 @@
 from stapi_fastapi.models.shared import HTTPException as HTTPExceptionModel
 from stapi_fastapi.models.shared import Link
 
-
-class StapiException(HTTPException):
-    def __init__(self, status_code: int, detail: str) -> None:
-        super().__init__(status_code, detail)
-
-
-class StapiRouter:
-    NAME_PREFIX = "stapi"
+"""
+/products/{component router} # router for each product added to main router
+/orders # list all orders
+"""
+class MainRouter:
+    NAME_PREFIX = "main"
     backend: StapiBackend
```

Comment:
The backend for this class is going to need to be different than is currently implemented. I believe only the `get_order()` method will need to use it. In other words, we need to allow implementations to define how to retrieve order records, but all other behaviors of this router class will not require customization per implementation.